pkg/testing/integration_test.go

```diff
@@ -61,6 +61,10 @@ type IntegrationSuite struct {
 	cancel    context.CancelFunc
 }
 
+const (
+	maxRetries = 10
```

Comment:
@viveksinghggits you might want to remove this?

Reply:
Hi @PrasadG193 ,
Yes, I have removed that.